src/api/workspaces/index.ts

```diff
@@ -49,6 +50,15 @@ export async function leaveWorkspace(workspaceId: string): Promise<boolean> {
   return (await api.callOld(MUTATION_LEAVE_WORKSPACE, { workspaceId })).leaveWorkspace;
 }
 
+/**
+ * Leave workspace
+ *
+ * @param {string} workspaceId - id of workspace to delete
+ */
+export async function deleteWorkspace(workspaceId: string): Promise<boolean> {
+  return (await api.callOld(MUTATION_DELETE_WORKSPACE, { workspaceId })).deleteWorkspace;
```

Comment:
please, use `api.call` instead of deprecated `api.callOld`. There is a little difference

Reply:
okay 👌